R/calc.dtsurvey.R

```diff
@@ -301,6 +302,23 @@ compute = function(DT, x, by = NULL, metrics, ci_method = 'mean', level = .95, t
     ndis_fun = NULL
   }
 
+  #vcov
```

Comment:
This new section seems great. 
However, the following section (~322-359) is a bit confusing to me. There is already commenting, but if you can think of additional commenting that would help explain the workings, that might be good. Admittedly, this is probably because you're using substitute and eval the way they are supposed to be used as opposed to my willy nilly way of using them, so it's in good measure my issue! This is especially the case since I know this is old code so I should have noticed this long ago.